submissions/fiyingabriel/css/style.css

```diff
@@ -0,0 +1,227 @@
+/* Global stylings */
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+ font-family: "Helvetica Neue", "Open Sans", Helvetica, Arial, sans-serif;
+ color: #5f5f5f;
+}
+img {
+  width: 100%;
+  height: auto;
+ }
+/* header */
+header {
+ background: #004b18;
+ display: flex;
+ flex-direction: column;
+ color: white;
+ padding: 1em;
+}
+header .logo{
+ width:10em;
+ margin-left: 0.2em;
+}
+header ul{
+ list-style: none;
+ margin-left: 0.2em;
+}
+header ul li{
+ padding: 0.6em;
+ margin-left: 2em;
+}
+header ul li a{
+ text-decoration: none;
+ color: white;
+}
+/* main */
+main{
+  padding: 2em;
+}
+main .heading{
+ width: 80%;
+  margin: 1em auto;
+ text-align: center ;
+}
+main .heading h1{
+ color: #004b18;
+}
+main .existed{
+ margin-bottom: 1em;
+ text-align: center;
+ font-size: 1rem;
+}
+main form{
```

Comment:
It is not advisable to style HTML element themself. give it a class name or id. What if you have form tag elsewhere on that same page?

same goes to other places where you are style HTML element. expect if you want that element to have that style generally. E.g removing the underline on an anchor tag, `<a>` tag.  or removing the default margin and padding a body tag has